### PR TITLE
Add FAQ section to Built-in CSS Support

### PR DESCRIPTION
**Pull request by rafaelalmeidatk**
I added a FAQ section to answer questions I see frequently. Motivation: https://github.com/vercel/next.js/discussions/13625#discussioncomment-23186

### CODE REVIEW

File: docs/basic-features/built-in-css-support.md [hunk 1]
```diff
@@ -200,6 +200,16 @@ export default HelloWorld
 
 Please see the [styled-jsx documentation](https://github.com/zeit/styled-jsx) for more examples.
 
+## FAQ
+
+### How can I import global styles from `node_modules` inside components code?
+
+Due to CSS ordering issues, all CSS files that aren't modules should be imported inside `pages/_app.js`.
```
Comment (rafaelalmeidatk) [inline docs/basic-features/built-in-css-support.md:207]: I am not 100% sure this is the reason 🤔

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 13s | 12.8s | -208ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.308 | 2.24 | -0.07 |
| / avg req/sec | 1083.3 | 1115.84 | ⚠️ +32.54 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.389 | 1.403 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1799.9 | 1781.98 | -17.92 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| index.html gzip | 927 B | 927 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 924 B | 924 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 16.1s | 15.4s | -765ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _error.js | 878 kB | 878 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 878 kB | 878 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>

File: docs/basic-features/built-in-css-support.md [hunk 2]
```diff
@@ -200,6 +200,16 @@ export default HelloWorld
...
+
+### Does it work with JavaScript disabled?
+
+Yes, if you disable JavaScript the CSS will still be loaded in the production build (`next start`). During development, we require JavaScript to be enabled to provide the best developer experience with Fast Refresh.
```
Comment (lfades) [inline docs/basic-features/built-in-css-support.md:211]:
```suggestion
Yes, if you disable JavaScript the CSS will still be loaded in the production build (`next start`). During development, we require JavaScript to be enabled to provide the best developer experience with [Fast Refresh](https://nextjs.org/blog/next-9-4#fast-refresh).
```

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 12.3s | 11.6s | -716ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.144 | 2.161 | ⚠️ +0.02 |
| / avg req/sec | 1166.11 | 1157.11 | -9 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.355 | 1.319 | -0.04 |
| /error-in-render avg req/sec | 1845.08 | 1895.69 | ⚠️ +50.61 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| index.html gzip | 927 B | 927 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 924 B | 924 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 12.6s | 12.9s | ⚠️ +270ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _error.js | 878 kB | 878 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 878 kB | 878 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>

**Comment by chibicode** [comment]
@rafaelalmeidatk Thanks! I think it might be better to move the section about `node_modules` right before the "Adding Component-Level CSS" section so it’s more visible. I edited it here: https://github.com/vercel/next.js/pull/13846/commits/88ee7341e742af2f7df9c0ffeddf05de0b1638d9 cc @lfades

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 13.1s | 13s | -21ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.229 | 2.302 | ⚠️ +0.07 |
| / avg req/sec | 1121.72 | 1085.83 | -35.89 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.427 | 1.51 | ⚠️ +0.08 |
| /error-in-render avg req/sec | 1751.42 | 1655.52 | -95.9 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| index.html gzip | 927 B | 927 B | ✓ |
| link.html gzip | 936 B | 936 B | ✓ |
| withRouter.html gzip | 924 B | 924 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| buildDuration | 14.3s | 14.1s | -158ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.61 kB | 6.61 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..9545.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57 kB | 57 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.69 kB | 5.69 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.5 kB | 52.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.21 kB | 8.21 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.32 kB | 5.32 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | rafaelalmeidatk/next.js patch-5 | Change |
| - | - | - | - |
| _error.js | 878 kB | 878 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 878 kB | 878 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>

**Comment by rafaelalmeidatk** [comment]
@chibicode I am fine with the change but I think it would be good to explain why it isn't possible yet since this is a common situation

**Review by Timer — APPROVED** [review]
LGTM!

**Pull request merged**